<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Body</class>
 <widget class="QDialog" name="Body">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(61,61,61);
	font-family: &quot;Trebuchet MS&quot;;
	color: rgb(174, 174, 174);
	}

#Titel {
	font-size: 12pt;
	background-color: #761811;
	font-weight: bold;
	padding: 1px;
	border-radius: 8px;
	color: white;
}

#pushButton_BMI{
	background-color:  #761811;
	border-radius: 5px;
}

#pushButton_Speichern{
	background-color: rgb(85,85,85);
	border: 2px solid rgb(43,43,43);
	border-radius: 5px;
   font-size: 6pt;
}

#pushButton_Back{
	background-color:  #761811;
	border-radius: 5px;
}

QdoubleSpinBox{
	color: rgb(174, 174, 174);
	background-color: rgb(59, 59, 59);
}</string>
  </property>
  <widget class="QLabel" name="Titel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Körperangaben</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Weight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gewicht:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Weight">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> kg</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Height">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Körpergröße:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Height">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> m</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Arm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oberarmumfang:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Arm">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Brust">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Brustumfang:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Brust">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Huefte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hüftumfang:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Huefte">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Oberschenkel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Oberschenkelumfang:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Oberschenkel">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Waden">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wadenumfang:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Waden">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string> cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_BMI">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>417</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>BMI berechnen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Speichern">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zurück</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
